### PR TITLE
Webhooks endpoints

### PR DESCRIPTION
**Pull request by AbdelilahLbardi**
Changelist

- [x] Add support for `GET` & `DELETE` to the `HttpLayer`.
- [x] `Webhooks` list endpoint.
- [x] `Webhooks` find endpoint.
- [x] `Webhooks` delete endpoint.
- [x] `Webhooks` create endpoint.
- [x] `Webhooks` update endpoint.
- [x] Documentation update
- [x] Changelog update

Breaking changes:
- N/A

### CODE REVIEW

**Review by fosron — APPROVED** [review]

**Comment by fosron** [comment]
@AbdelilahLbardi In README examples we should have `$mailersend = new MailerSend(['api_key' => 'key']);`

**Comment by AbdelilahLbardi** [comment]
@fosron Changes done!

**Review by fosron — APPROVED** [review]

File: src/Endpoints/Webhook.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+     */
+    public function find(string $id): array
+    {
+        GeneralHelpers::assert(
+            fn () => Assertion::minLength($id, 1, 'Webhook id is required.')
+        );
+
+        return $this->httpLayer->get(
+            $this->buildUri($this->endpoint . '/' . $id),
+            []
```
Comment (doobas) [inline src/Endpoints/Webhook.php:107]: Empty array can be described as default parameter in `get` method.

**Review by doobas — APPROVED** [review]

File: src/Endpoints/Webhook.php (hunk 1 above)
Comment (AbdelilahLbardi) [reply src/Endpoints/Webhook.php]: Done! Thanks

**Pull request merged**